docs/autodidax.md

```diff
@@ -1623,8 +1623,7 @@ def execute_compiled(compiled, out_avals, *args):
   return [handle_result(aval, buf) for aval, buf in zip(out_avals, out_bufs)]
 
 default_input_handler = xb.get_backend(None).buffer_from_pyval
-input_handlers = {ty: default_input_handler for ty in
-                  [bool, int, float, np.ndarray, np.float64, np.float32]}
+input_handlers = dict.fromkeys([bool, int, float, np.ndarray, np.float64, np.float32], default_input_handler)
```

Comment:
I didn't know about this one!

Reply:
Yeah I don't know if I agree with ruff on this one... maybe we should revert these and turn off this check? What do you think?